ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java

```diff
@@ -16,7 +16,7 @@
 
     private static final String URL = "/admin/index.htm";
 
-    private static final By MANAGE_USERS = By.cssSelector("#legacyui-mangeUsers a");
+    private static final By MANAGE_USERS = By.cssSelector("#legacyui-manageUsers a");
```

Comment:
this would fail if i merged it it, we can only check it again once we have legacy change merged and tell maven to replace -snapshopts

Reply:
Sure thanks Actually  Legacy ui change was merged here https://github.com/openmrs/openmrs-module-legacyui/pull/134/files , and lucky  enough we are running on the same version of legacy ui as seen https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/pom.xml#L67

Reply:
Hey @k-joseph  any suggestion about my comment

Reply:
Thanks  alot @k-joseph